src/lib/utilities/API/types.ts

```diff
@@ -83,5 +83,14 @@ export type PostPostResponseType = {
   name: string;
 }
 
+export type PostDirectionType = {
```

Comment:
it's better
```suggestion
export type DirectionType = {
```

`Post` refers to POST request when comparing with `GetDirectionType ` next ot it..